sdks/python/apache_beam/internal/apiclient.py

```diff
@@ -391,18 +394,38 @@ def stage_file(self, gcs_or_local_path, file_name, stream,
 
   # TODO(silviuc): Refactor so that retry logic can be applied.
   @retry.no_retries  # Using no_retries marks this as an integration point.
-  def create_job(self, job):
-    """Submits for remote execution a job described by the workflow proto."""
-    # Stage job resources and add an environment proto with their paths.
+  def create_job(self, job, pipeline_options):
+    """Creates a job description.
```

Comment:
Correctly format the multiline doc string

```
"""Creates a job description.

Additionally, it may stage it and/or submit it for remote execution.
"""
```